### PR TITLE
Add: Filament fees adapter 

### PR DESCRIPTION
**Pull request by rajeebkm**
- Daily fees and total fees added
- Daily and total trading volume added

### CODE REVIEW

**Comment by llamatester** [comment]
The filament adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts fees filament
    
    🦙 Running FILAMENT adapter 🦙
    ---------------------------------------------------
    1727256001
    1727169601
    Start Date:	Tue, 24 Sep 2024 00:00:00 GMT
    End Date:	Wed, 25 Sep 2024 00:00:00 GMT
    ---------------------------------------------------
    
    1727256001
    1727169601
    SEI 👇
    Backfill start time: 7/9/2024
    Daily fees: 2
    Total fees: 14

**Comment by llamatester** [comment]
The filament adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs filament
    
    🦙 Running FILAMENT adapter 🦙
    ---------------------------------------------------
    Start Date:	Tue, 24 Sep 2024 00:00:00 GMT
    End Date:	Wed, 25 Sep 2024 00:00:00 GMT
    ---------------------------------------------------
    
    TypeError: Cannot destructure property 'allTimeVolume' of 'res.data' as it is undefined.
        at fetch (/home/runner/work/dimension-adapters/dimension-adapters/dexs/filament/index.ts:10:11)
        at processTicksAndRejections (node:internal/process/task_queues:95:5)
        at async getChainResult (/home/runner/work/dimension-adapters/dimension-adapters/adapters/utils/runAdapter.ts:49:18)
        at async Promise.all (index 0)
        at async runAdapter (/home/runner/work/dimension-adapters/dimension-adapters/adapters/utils/runAdapter.ts:27:20)
        at async /home/runner/work/dimension-adapters/dimension-adapters/cli/testAdapter.ts:75:21 {
      chain: 'near'
    }

**Comment by llamatester** [comment]
The filament adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts fees filament
    
    🦙 Running FILAMENT adapter 🦙
    ---------------------------------------------------
    1727266213
    1727179813
    Start Date:	Tue, 24 Sep 2024 00:00:00 GMT
    End Date:	Wed, 25 Sep 2024 00:00:00 GMT
    ---------------------------------------------------
    
    1727266213
    1727179813
    SEI 👇
    Backfill start time: 7/9/2024
    Daily fees: 2
    Total fees: 14

**Comment by llamatester** [comment]
The filament adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs filament
    
    🦙 Running FILAMENT adapter 🦙
    ---------------------------------------------------
    Start Date:	Tue, 24 Sep 2024 00:00:00 GMT
    End Date:	Wed, 25 Sep 2024 00:00:00 GMT
    ---------------------------------------------------
    
    NEAR 👇
    Backfill start time: 10/3/1973
    NO METHODOLOGY SPECIFIED
    End timestamp: 1727222400 (2024-09-25T00:00:00.000Z)
    Daily volume: 225
    Total volume: 274.36 k

File: dexs/filament/index.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  return {
+    timestamp,
+    dailyVolume: volumeIn24Hours,
+    totalVolume: allTimeVolume,
+  };
+};
+
+const adapter: SimpleAdapter = {
+  adapter: {
+    near: {
```
Comment (dtmkeng) [inline dexs/filament/index.ts]: why fees is  sei but dexs volume is `near` chain ?

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Review by Rezacb0413 — APPROVED** [review]

**Comment by llamatester** [comment]
The filament adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs filament
    
    🦙 Running FILAMENT adapter 🦙
    ---------------------------------------------------
    Start Date:	Thu, 26 Sep 2024 00:00:00 GMT
    End Date:	Fri, 27 Sep 2024 00:00:00 GMT
    ---------------------------------------------------
    
    SEI 👇
    Backfill start time: 10/3/1973
    NO METHODOLOGY SPECIFIED
    End timestamp: 1727395200 (2024-09-27T00:00:00.000Z)
    Daily volume: 400
    Total volume: 275.07 k

**Comment by llamatester** [comment]
The filament adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts fees filament
    
    🦙 Running FILAMENT adapter 🦙
    ---------------------------------------------------
    Start Date:	Thu, 26 Sep 2024 00:00:00 GMT
    End Date:	Fri, 27 Sep 2024 00:00:00 GMT
    ---------------------------------------------------
    
    1727423951
    1727337551
    SEI 👇
    Backfill start time: 7/9/2024
    Daily fees: 5
    Total fees: 14

File: dexs/filament/index.ts (hunk 1 above)
Comment (rajeebkm) [reply dexs/filament/index.ts]: I was using chain from [[CHAIN.SEI] in both fees and volume.

I have changed to `sei` in case of volume now.

**Comment by rajeebkm** [comment]
hi @dtmkeng, fees section isn't visible on defiLlama dashboard for Filament Finance.

**Pull request merged**